section3/reflection.md

```diff
@@ -1,13 +1,28 @@
 ## Section 3 Reflection
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
-
+* I am not going to lie, it was a bit uncomfortable but also expected. I did spend time looking through past exercises, lessons, and re-watched some videos that were in our section plan. Lots of Google searches as well. I am glad I started section 3 after our last lesson and I found my notes to be valuable again to me.
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
-
+* I think Instagram uses Arrays to store information. Instagram might use Arrays to store information on how many followers a user might have on their profile, to how many posts a user might have, or how many likes a post has.
 3. In your own words, explain what an Array method is.
-
+* Array methods are tools(or is the technical term operators?) that help developers calculate or change a function in an Array. It can save a lot of time from doing manual work.
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
-
+* The purpose of a loop is to repeat a task based on a condition, until that condition returns false. Loops can be helpful in arrays, especially if an array has a ton of elements in it, a loop can access all the elements in an array. Or sometimes a loop can just access **some** elements (like odd/even numbers), which again, is especially helpful if you have a lot of elements in a array.
 5. If you had to teach a 5 year old about loops, how would you explain it?
+* I guess I would say:
+ "Imagine that you had a hundred toys all over your room and your mom is asking you to clean up your room and to organize your toys from smallest to biggest to put on a shelf, thats going to take you forever, right? But imagine if you had robot named Loopy and you can just tell your robot that you want to organize your toys from smallest to biggest and your robot will make sure its perfect, that would be so helpful, right? Loopy can save you time comparing all the sizes of your toys and will make sure its all counted perfectly"
 
+ - And, that was the best I could come up with, I am sure there are way better examples!
 6. What questions do you still have about Arrays and loops?
+* Honestly I wasn't sure what was the best solution for the problem below and I Googled to see if there was a better array method and I felt a little lost in my searches. This ended up working in my terminal but I am wondering if there was a better way to execute this?
+
+- YOU DO: Write code that will add a new animal (type of your choice) to position 3.
+```
+animals.push('Llama');
+animals[0]= 'Zebra'
+animals[1]='Giraffe'
+animals[2]= 'Llama';
+animals[3]= 'Elephant';
```

Comment:
This is the line of code that answers the prompt `Write code that will add a new animal (type of your choice) to position 3.`! The others are ok to be there, just not necessary.

Reply:
And there are other ways, but I would say this is the most straightforward or "path of least resistance" (which is something we usually strive for in coding - don't make it overcomplicated if we don't need to!)